demo/admin/src/predefinedPage/EditPredefinedPage.tsx

```diff
@@ -0,0 +1,139 @@
+import { FormattedMessage } from "react-intl";
+
+const getQuery = gql`
+    query PredefinedPage($id: ID!) {
+        page: pageTreeNode(id: $id) {
+            id
+            name
+            slug
+            parentId
+            document {
+                __typename
+                ... on DocumentInterface {
+                    id
+                    updatedAt
+                }
+                ... on PredefinedPage {
+                    id
+                    type
+                }
+            }
+        }
+    }
+`;
+
+const updateMutation = gql`
+    mutation UpdatePredefinedPage($pageId: ID!, $input: PredefinedPageInput!, $lastUpdatedAt: DateTime, $attachedPageTreeNodeId: ID) {
+        savePredefinedPage(id: $pageId, input: $input, lastUpdatedAt: $lastUpdatedAt, attachedPageTreeNodeId: $attachedPageTreeNodeId) {
+            id
+            type
+        }
+    }
+`;
+
+interface Props {
+    id: string;
+}
+
+export const EditPredefinedPage: React.FC<Props> = ({ id }) => {
+    const stackApi = useStackApi();
+
+    const { data, loading } = useQuery<GQLPredefinedPageQuery, GQLPredefinedPageQueryVariables>(getQuery, {
+        variables: { id },
+    });
+
+    const [mutation] = useMutation<GQLUpdatePredefinedPageMutation, GQLUpdatePredefinedPageMutationVariables>(updateMutation, {
+        refetchQueries: ["PredefinedPage"],
+    });
+
+    if (loading) {
+        return <CircularProgress />;
+    }
+
+    return (
+        <FinalForm
+            mode="edit"
+            initialValues={{ type: data?.page?.document?.__typename === "PredefinedPage" ? data.page.document.type : undefined }}
+            onSubmit={() => {
+                mutation({ variables: { pageId: id, input: { type: "News" }, attachedPageTreeNodeId: id } });
+            }}
+        >
+            {({ pristine, hasValidationErrors, submitting, handleSubmit, hasSubmitErrors }) => {
+                return (
+                    <EditPageLayout>
+                        <Toolbar>
+                            <ToolbarItem>
+                                <IconButton onClick={stackApi?.goBack}>
+                                    <ArrowLeft />
+                                </IconButton>
+                            </ToolbarItem>
+                            <PageName pageId={id} />
+                            <ToolbarFillSpace />
+                            <ToolbarItem>
+                                <SplitButton disabled={pristine || hasValidationErrors || submitting}>
+                                    <SaveButton color="primary" variant="contained" hasErrors={hasSubmitErrors} type="submit">
+                                        <FormattedMessage id="comet.generic.save" defaultMessage="Save" />
+                                    </SaveButton>
+                                    <SaveButton
+                                        color="primary"
+                                        variant="contained"
```

Comment:
`color`, `variant` and the "Save" text are already set to these values by default.

```suggestion
                                    <SaveButton hasErrors={hasSubmitErrors} type="submit" />
                                    <SaveButton
```